python/trainers/ppo_trainer.py

```diff
@@ -185,22 +184,16 @@ def take_action(self, info):
             feed_dict[self.model.new_mean] = new_mean
             feed_dict[self.model.new_variance] = new_variance
             run_list = run_list + [self.model.update_mean, self.model.update_variance]
-            # only ask for memories if use_recurrent
-            if self.use_recurrent:
-                actions, a_dist, value, ent, learn_rate, memories, _, _ = self.sess.run(run_list, feed_dict=feed_dict)
-            else:
-                actions, a_dist, value, ent, learn_rate, _, _ = self.sess.run(run_list, feed_dict=feed_dict)
-                memories = None
+
+        values = self.sess.run(run_list, feed_dict=feed_dict)
+        run_out = dict(zip(run_list, values))
+        self.stats['value_estimate'].append(run_out[self.model.value])
```

Comment:
This needs to be 
`self.stats['value_estimate'].append(run_out[self.model.value].mean())`
Made a PR but it will confict